feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go

```diff
@@ -60,6 +63,9 @@ func TestLinecardPowerAdmin(t *testing.T) {
 
 func TestControllerCardPowerAdmin(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
+	if deviations.PlatformPowerDownUpSkip(dut) {
```

Comment:
Could you please create separate deviations for fabric and controller card tests?
We'd like to reuse just one of those

Reply:
Sure